lib/std/zig/system/NativePaths.zig

```diff
@@ -188,6 +191,22 @@ fn deinitArray(array: *ArrayList([:0]u8)) void {
     array.deinit();
 }
 
+pub inline fn getIncludeDirs(self: *const NativePaths) []const [:0]const u8 {
+    return self.include_dirs.items;
+}
+
+pub inline fn getLibDirs(self: *const NativePaths) []const [:0]const u8 {
+    return self.lib_dirs.items;
+}
+
+pub inline fn getFrameworkDirs(self: *const NativePaths) []const [:0]const u8 {
+    return self.framework_dirs.items;
+}
+
+pub inline fn getRpaths(self: *const NativePaths) []const [:0]const u8 {
+    return self.rpaths.items;
+}
+
```

Comment:
why `inline`?

why do these getters even exist?

Reply:
It made sense to me instead of typing `paths.lib_dirs.items` everywhere but it's a matter of preference so I am happy to remove it.